src/pages/index.tsx

```diff
@@ -0,0 +1,6 @@
+import React from "react";
+import { Redirect } from "@docusaurus/router";
+
+export default function Home() {
+  return <Redirect to="how-to-play-online-werewolf" />;
```

Comment:
Couldn't find a way to have two slugs for a doc, so added this redirect in for the guide home page.